src/modules/themes/themes/GoldDark.tsx

```diff
@@ -43,6 +43,8 @@ export const GoldDark = {
 
       border: '#474d57',
 
+      wallet: '#3B99FC',
```

Comment:
We can leave this color inside the SVG file to avoid adding it to the Honeycomb theme.